### ISSUE TITLE
Slide changed eventData new/old index error

### ISSUE THREAD

**Issue by toddanglin** [TheOriginalJosh/nativescript-slides#67]
I started working with the `changed` event for the `SlideContainer`, and while debugging I noticed that the `newIndex` and `oldIndex` properties on the event's `eventData` property seem to be reversed when using the `nextSlide` method to navigate.

When navigating from the first slide to the second slide (using "nextSlide" method), the "change" event fires, and if you inspect the `eventData` you see:

```
newIndex: 0
oldIndex: 1
```

You *should* see `newIndex: 1, oldIndex: 0`. And this continues as you move through the slides (`newIndex` and `oldIndex` are reversed).

**NOTE:** This does NOT happen when swiping to navigate between slides. When swiping, the new/old indexes are CORRECT. It only seems to happen when using the `nextSlide` method to programmatically navigate.

Given the difference in behavior from swiping and navigating pragmatically, I assume there is just some kind of race condition.

**Comment by toddanglin** [comment]
I took a look at the code, and I *think* that the solution could be as simple as reversing these two lines of code in the `nextSlide`/`previousSlide` methods:

**CURRENTLY:**
```
this.showRightSlide(this.currentPanel).then(() => {
   this.triggerChangeEventRightToLeft();
   this.setupPanel(this.currentPanel.right);
});
```
**SHOULD BE?**
```
this.showRightSlide(this.currentPanel).then(() => {
   this.setupPanel(this.currentPanel.right);
   this.triggerChangeEventRightToLeft();
});
```
The `applySwipe` method calls these methods in that order (`setup` and then `trigger`). Not sure if there is a reason they are reversed in the `nextSlide` and `previousSlide` methods.

I tried this locally and it seems to fix the problem. I'll go ahead and create a quick pull request. If there are no side effects I'm missing, feel free to merge and version bump. :)
